### ISSUE TITLE
v1.6.1 IAM permissions not fully documented for instance role and logging

### ISSUE THREAD

**Issue by chrisdag** [aws/aws-parallelcluster#621]
Hi folks!

Minor doc/policy issue to file here ... 

Running brand new cfncluster 1.6.1 in a python virtualenv with IAM policies set up exactly as documented on: https://cfncluster.readthedocs.io/en/latest/iam.html

But still see this error on the role associated with the compute fleet members:

```
IAM role error on user provided role ec2-cfncluster-ICL-HPC-role: action logs:* is implicitDeny
See https://cfncluster.readthedocs.io/en/latest/iam.html
```

Note that https://cfncluster.readthedocs.io/en/latest/iam.html has zero IAM permissions related to "logs:" or cloudwatch logging at all in the section documenting the CfnClusterInstancePolicy.

**Comment by sean-smith** [comment]
Thanks for reporting this, there's no cloudwatch log permissions needed in 1.6.1. We removed cloudwatch logs as a dependency in 1.6.0. You can go ahead and [turn off config sanity](https://cfncluster.readthedocs.io/en/latest/configuration.html#sanity-check) until we fix the validation.

```
sanity_check = false
```

**Comment by sean-smith** [comment]
This was fixed in `2.1.0` https://github.com/aws/aws-parallelcluster/commit/bdc5564cad864a88459d0355451e2035c2bb924f
